Command/ProcessEventCommand.php

```diff
@@ -40,6 +40,11 @@ class ProcessEventCommand extends ContainerAwareCommand
     private $channels;
 
     /**
+     * @var int
+     */
+    private $processedEventsCount = 0;
```

Comment:
This variable should be local. The limit is per channel. The command can run multiple channel on the same run.